scripts/prove_rpc.sh

```diff
@@ -97,6 +97,7 @@ if [[ $8 == "test_only" ]]; then
     echo "Proving blocks from ($START_BLOCK) to ($END_BLOCK)"
     command="cargo r --release --package zero --bin leader -- \
 --test-only \
+--use-test-config \
```

Comment:
this step is `test-only`, i.e. no proving but only witness generation so `--use-test-config` has no effect

Reply:
Done.